scripts/vet-proto.sh

```diff
@@ -23,12 +23,9 @@ if [[ "$1" = "-install" ]]; then
   if [[ "${GITHUB_ACTIONS}" = "true" ]]; then
     PROTOBUF_VERSION=25.2 # Shows up in pb.go files as v4.22.0
     PROTOC_FILENAME=protoc-${PROTOBUF_VERSION}-linux-x86_64.zip
-    pushd /home/runner/go
-      wget https://github.com/google/protobuf/releases/download/v${PROTOBUF_VERSION}/${PROTOC_FILENAME}
-      unzip ${PROTOC_FILENAME}
-      protoc --version # Check that the binary works.
-    popd
-  else 
+    chmod +x ./scripts/install_protoc.sh
```

Comment:
This should be done locally on the file and then pushed to the branch :)

Reply:
Updated